docs/en/log_management.rst

```diff
@@ -0,0 +1,49 @@
+.. include:: ../common/common_definitions.rst
+
+.. _Log_Management:
+
+Retention Policy
+================
+
+Log management of a OP and an RP
+------------------------------------
+
+OPs and RPs MUST retain the following. 
+
+1. A transaction log containing the exchanged messages. The messages stored in the log MUST be at least the following:
+
+    - **AuthenticationRequest**
+    - **AuthenticationResponse** related to the *AuthenticationRequest**.
+    - **TokenRequest** related to the *AuthenticationRequest*.
+    - **TokenResponse** related to the *TokenRequest*. 
+    - The **UserInfoRequest** related to the *TokenRequest*, if any.
+    - The **UserInfoResponse** related to the *UserInfoRequest*, if any.
+    - **RevocationRequest** related to the *TokenRequest*, if any.
+    - The **RevocationResponse** related to the *RevocationRequest*, if any.
+
+2. A federation log containing, for each *AuthenticationRequest*, the **Trust Chain** related to the entity which messages are being exchanged with. In the case of a log handled by an OP receiving an authentication request from an RP, it MUST contain:
+
+    - The **Entity Configuration** of the requesting RP.
+    - The **Entity Configuration** of the SA (in the case of an aggregated RP), if any.
```

Comment:
~The **Entity Configuration** of the SA (in the case of an aggregated RP), if any.~

not required, the Trust Chain only have the Leaf's EC and the TA's EC

the intermediary EC are not required, so they are not required in the retention of the trust chain

Reply:
ok